urs-dev/requirements.yaml

```diff
@@ -1,4 +1,10 @@
 dependencies:
   - name: cassandra
     repository: https://charts.bitnami.com/bitnami
-    version: 9.1.3
\ No newline at end of file
+    version: 9.1.3
+    heap:
```

Comment:
Remove this and add the values for Cassandra in values.yml. Also you can see the variables name from [this](https://artifacthub.io/packages/helm/bitnami/cassandra) URL.

Reply:
```yaml
# Default values for urs-dev.
# This is a YAML-formatted file.
# Declare variables to be passed into your templates.

connector_main:
  enabled: true
  name: connector-main
  image: codegb/urs-connector-main
  cassandra:
    keyspace_name: url-space
    contact_point: 0.0.0.0
    port: 9042

common:
  image_pull_policy: Always
  memory:
    microservices: "512M"

cassandra:
  image:
    debug: true
  jvm:
    maxHeapSize: 500M
    newHeapSize: 200M
```